packages/hint-x-content-type-options/src/hint.ts

```diff
@@ -63,7 +63,12 @@ export default class XContentTypeOptionsHint implements IHint {
              */
 
             if (nodeName === 'script' ||
-                (nodeName === 'link' && normalizeString(element.getAttribute('rel')) === 'stylesheet')) {
+                (nodeName === 'link' &&
+                    (normalizeString(element.getAttribute('rel'), ''))
```

Comment:
I think you should move this to an `isStylesheet` method.

Reply:
@sarvaje I refactored the code a bit.

Reply:
@alrra I like the refactor :)